wolfssl/wolfcrypt/settings.h

```diff
@@ -216,6 +219,23 @@
     #include <nx_api.h>
 #endif
 
+#if defined(WOLFSSL_ESPIDF)
+    #define FREERTOS
+    #define WOLFSSL_LWIP
+    #define NO_WRITEV
+    #define SIZEOF_LONG_LONG 8
+    #define NO_WOLFSSL_DIR
+    #define WOLFSSL_NO_CURRDIR
+
+    #define TFM_TIMING_RESISTANT
+    #define ECC_TIMING_RESISTANT
+    #define WC_RSA_BLINDING
+#if !defined(WOLFSSL_USER_SETTINGS)
+    #define WOLFSSL_ESPWROOM32
+    #define HAVE_ECC
```

Comment:
I'm not sure we want to force all ESP users to have ECC enabled by default.  Can we remove this and put it in our example user_settings.h instead?

Reply:
Disregard, based on conversation.